src/components/NFTCollection/CollectionItem.test.tsx

```diff
@@ -0,0 +1,59 @@
+import { render, fireEvent, screen  } from '@testing-library/react'
+import { Link } from '@imtbl/imx-sdk'
+import CollectionItem from './CollectionItem'
+
+const testProps = {
+    name: 'collectionName',
+    url: 'http.test.com',
+    price: 'price',
+    orderId: 'orderId',
+}
+
+const linkAddress = 'https://link.ropsten.x.immutable.com';
+const link = new Link(linkAddress);
+
+describe('CollectionItem component', () => {
+    test('match snapshot', () => {
+        const { container } = render(
+            <CollectionItem
+                name={testProps.name}
+                url={testProps.url}
+                price={testProps.price}
+                orderId={testProps.orderId} />
+        );
+        expect(container).toMatchSnapshot()
+    })
+
+    test('Load placeholder image on error', () => {
+        render(
+            <CollectionItem
+                name={testProps.name}
+                url={testProps.url}
+                price={testProps.price}
+                orderId={testProps.orderId} />
+        );
+        const image = screen.getByRole('img');
+        fireEvent.error(image)
+        expect(image).toHaveAttribute('src','placeholderImg.jpg');
+    })
+
+    test('buyItem errors correctly', async () => {
```

Comment:
not clear what you meant

Reply:
Yep, I meant to say: 'trigger console log if buyItem times out'